components/ConvertSteps/ManageStep.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useCallback, useState } from 'react'
+import PropTypes from 'prop-types'
+import Step from './Step'
+
+function ManageStep({
+  title,
+  number,
+  handleTx,
+  active,
+  onSuccess,
+  onError,
+  onHashCreation,
+  retry,
+}) {
+  const [stepStatus, setStepStatus] = useState('waiting')
+  const [hasBeenActivated, setHasBeenActivated] = useState(false)
+  const [retrying, setRetrying] = useState(false)
+  const [hash, setHash] = useState()
+
+  const handleStepSigning = useCallback(async () => {
```

Comment:
This callback has two `await`s, so I'm wondering—is there any chance this component will get unmounted or re-rendered while it's waiting?

Reply:
I'll update this code to avoid a leak just in case.